docs/en/protocols/README.md

```diff
@@ -1,15 +1,10 @@
-# Probe Protocol
-It includes descriptions and definitions on how agents send collected metrics, logs, traces and events, as well as the format of each entity.
-
-## Probe Protocols
-They also related to the probe group. For more information, see [Concepts and Designs](../concepts-and-designs/overview.md).
-These groups are **language-based native agent protocol**, **service mesh protocol** and **3rd-party instrument protocol**.
```

Comment:
@wingwong-knh I directly removed this. We don't need this concept of group here anymore.